package.json

```diff
@@ -25,6 +25,7 @@
     "axios": "^0.27.2",
     "date-fns": "^2.29.3",
     "fuse.js": "^6.6.2",
+    "phosphor-react": "^1.4.1",
```

Comment:
Vi que instalou a lib de ícones, mas tb adicionou alguns ícones manualmente no projeto. Eles não estão nessa lib? Se estiverem, não seria mais interessante importar dela ao invés de baixar?

Reply:
Atualizei a branch para não dar conflitos, acho que a instalação da lib foi feita por outro colega.

Reply:
Show, nesse caso eu sugiro que quando for usar ícones dar uma olhada se não tem nessa lib, pra evitar adicionar mais arquivos estáticos no projeto. E caso não tenha, conversar com o designer para ver se não seria interessante padronizar esses ícones de alguma fonte (como o phosphor)

Reply:
@AdsonAmorim Que bom que você conseguiu participar da reunião hoje, a questão do design é algo que estamos dialogando já algum tempo para poder padronizar